<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="title_label">
     <property name="text">
      <string>Settings</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="path_layout">
     <item>
      <widget class="QLabel" name="path_label">
       <property name="text">
        <string>Library Folder:</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btn_select_folder">
       <property name="text">
        <string>Select Folder</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="btn_scan">
     <property name="text">
      <string>Scan Library Now</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="chk_auto_scan">
     <property name="text">
      <string>Automatically scan library on startup</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="api_label">
     <property name="text">
      <string>API Settings</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="token_layout">
     <item>
      <widget class="QLineEdit" name="token_input">
       <property name="placeholderText">
        <string>Anilist Access Token</string>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_save_token">
       <property name="text">
        <string>Save Token</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="data_label">
     <property name="text">
      <string>Data Management</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btn_clear_covers">
     <property name="text">
      <string>Clear Cover Cache</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btn_clear_db">
     <property name="text">
      <string>Clear Library Database</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
